### ISSUE TITLE
An error on Mac OS Sierra

### ISSUE THREAD

**Issue by yoshiyama** [IntelVCL/Open3D#691]
An Error as below is happened when rgbd_redwood.py is executed.
Let me know how to solve it.
My computer environment is 

- OS: MacOS Sierra version 10.12.1
- Python 3.6
- Open3D was installed using "conda install -c open3d-admin open3d" as below site.
- https://anaconda.org/open3d-admin/open3d

------ Error Report------
(py36) bash-3.2$ python rgbd_redwood.py 

Read Redwood dataset
RGBDImage of size 
Color image : 640x480, with 1 channels.
Depth image : 640x480, with 1 channels.
Use numpy.asarray to access buffer data.
2018-11-13 17:05:37.673 python[4421:419850] -[NSApplication _setup:]: unrecognized selector sent to instance 0x7fd7e69b74e0
2018-11-13 17:05:37.729 python[4421:419850] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSApplication _setup:]: unrecognized selector sent to instance 0x7fd7e69b74e0'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff8512a48b __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x00007fff99a31cad objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff851abc94 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   CoreFoundation                      0x00007fff8509cb55 ___forwarding___ + 1061
	4   CoreFoundation                      0x00007fff8509c6a8 _CF_forwarding_prep_0 + 120
	5   libtk8.6.dylib                      0x000000011bcc031d TkpInit + 413
	6   libtk8.6.dylib                      0x000000011bc1817e Initialize + 2622
	7   _tkinter.cpython-36m-darwin.so      0x0000000118fdea16 _tkinter_create + 1174
	8   python                              0x0000000107c38088 _PyCFunction_FastCallDict + 200
	9   python                              0x0000000107d0ef4f call_function + 143
	10  python                              0x0000000107d0cabf _PyEval_EvalFrameDefault + 46847
	11  python                              0x0000000107d00209 _PyEval_EvalCodeWithName + 425
	12  python                              0x0000000107d0fb1c _PyFunction_FastCallDict + 364
	13  python                              0x0000000107bb68b0 _PyObject_FastCallDict + 320
	14  python                              0x0000000107bddfe8 method_call + 136
	15  python                              0x0000000107bbdefe PyObject_Call + 62
	16  python                              0x0000000107c5f385 slot_tp_init + 117
	17  python                              0x0000000107c638c1 type_call + 241
	18  python                              0x0000000107bb6821 _PyObject_FastCallDict + 177
	19  python                              0x0000000107bbea67 _PyObject_FastCallKeywords + 327
	20  python                              0x0000000107d0f048 call_function + 392
	21  python                              0x0000000107d0cb6f _PyEval_EvalFrameDefault + 47023
	22  python                              0x0000000107d0f30c fast_function + 188
	23  python                              0x0000000107d0efac call_function + 236
	24  python                              0x0000000107d0cabf _PyEval_EvalFrameDefault + 46847
	25  python                              0x0000000107d00209 _PyEval_EvalCodeWithName + 425
	26  python                              0x0000000107d0fb1c _PyFunction_FastCallDict + 364
	27  python                              0x0000000107bb68b0 _PyObject_FastCallDict + 320
	28  python                              0x0000000107bddfe8 method_call + 136
	29  python                              0x0000000107bbdefe PyObject_Call + 62
	30  python                              0x0000000107d0ccc0 _PyEval_EvalFrameDefault + 47360
	31  python                              0x0000000107d00209 _PyEval_EvalCodeWithName + 425
	32  python                              0x0000000107d0f3ba fast_function + 362
	33  python                              0x0000000107d0efac call_function + 236
	34  python                              0x0000000107d0cabf _PyEval_EvalFrameDefault + 46847
	35  python                              0x0000000107d0f30c fast_function + 188
	36  python                              0x0000000107d0efac call_function + 236
	37  python                              0x0000000107d0cabf _PyEval_EvalFrameDefault + 46847
	38  python                              0x0000000107d00209 _PyEval_EvalCodeWithName + 425
	39  python                              0x0000000107d0f3ba fast_function + 362
	40  python                              0x0000000107d0efac call_function + 236
	41  python                              0x0000000107d0cabf _PyEval_EvalFrameDefault + 46847
	42  python                              0x0000000107d00209 _PyEval_EvalCodeWithName + 425
	43  python                              0x0000000107d58d4c PyRun_FileExFlags + 252
	44  python                              0x0000000107d58224 PyRun_SimpleFileExFlags + 372
	45  python                              0x0000000107d7ed66 Py_Main + 3734
	46  python                              0x0000000107bae929 main + 313
	47  libdyld.dylib                       0x00007fff9a310255 start + 1
	48  ???                                 0x0000000000000002 0x0 + 2
)
libc++abi.dylib: terminating with uncaught exception of type NSException
Abort trap: 6

**Comment by syncle** [comment]
I haven't seen this type of error so far. Here is my guess - Did the script crash before showing this window?
![image](https://user-images.githubusercontent.com/18297113/48400715-f7895900-e6db-11e8-9fb5-f39c3fd86d17.png)
If it is, the issue may comes from ``matplotlib``.

**Comment by yoshiyama** [comment]
Yes, it did.
matplotlib(version 3.0.1) is already installed using conda.

**Comment by yxlao** [comment]
@yoshiyama could you narrow down which line of python code caused the crash? 
As pointed out by @syncle , it could be related to matplotlib, this looks similar to your error message: https://stackoverflow.com/questions/32019556/matplotlib-crashing-tkinter-application/34109240#34109240

**Comment by syncle** [comment]
Yes. Try to comment lines relevant to the matplotlib. The more important thing is to see Visualizer window titled "Open3D". If matplotlib makes issue, I would recommend uninstall/install.

**Comment by yoshiyama** [comment]
Thank you for your advises.
It works by setting backend as Tkagg.

#backend :Agg is changed to 
backend:Tkagg 
on matplotlibrc file.

How to find where the matplotlibrc file is below command:
 python -c "import matplotlib;print(matplotlib.matplotlib_fname())"

Thank you for your cooperation.
